### PR TITLE
LPS-114788 | Master

### PR DESCRIPTION
**Pull request by Hanlf**
https://issues.liferay.com/browse/LPS-114788

### CODE REVIEW

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Message.path [hunk 1]
```diff
@@ -270,6 +270,11 @@
 	<td>//div[@role='alert' and contains(.,'Please select a valid date range.')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ERROR_MULTIPLE_UPLOAD_VALID</td>
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Message.path:274]: valid or invalid?

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Message.path (hunk 1 above)
Comment (austinchiang) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Message.path:274]: it's not clear what this path is trying to assert. can you change the name @Hanlf ?

File: portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro [hunk 2]
```diff
@@ -534,9 +534,31 @@ definition {
 			locator1 = "Button#PUBLISH",
 			value1 = "Publish");
 
-		AssertElementPresent.pauseAssertVisible(locator1 = "DocumentsAndMedia#BREADCRUMB_HOME");
+		if (isSet(validationError)) {
+			var list1 = ListUtil.newListFromString("${dmDocumentFileFailed}");
+			var list2 = ListUtil.newListFromString("${validationError}");
+			var size = ListUtil.size("${list1}");
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro:540]: @Hanlf please remember to use a space when order matters. This is happened a few times before

File: portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro [hunk 3]
```diff
@@ -534,9 +534,31 @@ definition {
...
+				var dmDocumentFileFailed = ListUtil.get("${list1}", "${i}");
+				var validationError = ListUtil.get("${list2}", "${i}");
+
+				AssertTextEquals.assertPartialText(
+					dmDocumentFile = "${dmDocumentFileFailed}",
+					key_alertMessage = "${validationError}",
+					locator1 = "Message#ERROR_MULTIPLE_UPLOAD_VALID",
+					value1 = "${validationError}");
+
+				var i = ${i} + 1;
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro:553]: same here

**Comment by austinchiang** [comment]
@Hanlf  please fix and resend

File: portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro (hunk 2 above)
Comment (Hanlf) [reply portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro:540]: @austinchiang The SF job will fail with a space here...

**Pull request closed without merging**